### PR TITLE
Unifies createDashboard functions

### PR DESCRIPTION
**Pull request by alonsogarciapablo**
This PR removes internal createDashboard and moves that login in to the public createDashboard :)

### CODE REVIEW

File: doc/quickstart.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ Here's how to create a dashboard:
 ```js
 cartodb.deepInsights.createDashboard('#dashboard', vizJSONurl, {}, function(err, dashboard) {
     // update a dashboard widget
-    var formulaWidget = dashboard.getWidget('whatever-id')
+    var formulaWidget = dashboard.widgets.get('whatever-id')
```
Comment (javisantana) [inline doc/quickstart.md]: not agree with this, let's keep it simple, having objects to access is harder than having a method.

Comment (alonsogarciapablo) [inline doc/quickstart.md]: No problem. I will leave this as it was before.

**Comment by alonsogarciapablo** [comment]
@javisantana I removed changes to the API from this PR, so it only includes some minor refactoring. Would you please take a look? Thanks!

**Comment by javisantana** [comment]
:es:

**Pull request merged**